test/NuGet.CommandLine.Staging.UnitTests/StagingCommandUnitTests.cs

```diff
@@ -39,7 +41,7 @@ public void StageCommand_WhenSourceNotProvidedErrorIsShown()
         public void StageCommand_WhenApiKeyNotProvidedErrorIsShown()
         {
             // Act
-            string[] args = { "stage", "-create", "abc", "-Source", "http://api.nuget.org/v3/index.json" };
+            string[] args = { "stage", "create", "abc", "-Source", "http://api.nuget.org/v3/index.json" };

```

Comment:
doesn't matter since this param is not used in the scenario

---
In reply to: [74158875](https://github.com/NuGet/NuGet.Staging/pull/68#discussion_r74158875) [](ancestors = 74158875)